<ui version="4.0" >
 <class>frmProgress</class>
 <widget class="QWidget" name="frmProgress" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>580</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>580</width>
    <height>100</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>580</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>QNapi - pobieranie napisów...</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../src/resources.qrc" >:/icon/qnapi-22-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="lbFileName" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbAction" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Pobieranie napisów do filmu...</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QProgressBar" name="pbProgress" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value" >
        <number>0</number>
       </property>
       <property name="alignment" >
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="textVisible" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../src/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>frmProgress</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>534</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel" >
     <x>289</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
